custom_components/pyscript/__init__.py

```diff
@@ -21,7 +21,8 @@
     EVENT_STATE_CHANGED,
     SERVICE_RELOAD,
 )
-from homeassistant.core import Config, Event as HAEvent, HomeAssistant, ServiceCall
+from homeassistant.core import Event as HAEvent, HomeAssistant, ServiceCall
+from homeassistant.core_config import Config
```

Comment:
The second parameter to `async_setup` isn't an instance of `Config`. Please use `dict[str, Any]` or `ConfigType` instead.

Reply:
Good call, changed and tested locally.  PR tests now pass as well.